_layouts/default.html

```diff
@@ -36,8 +36,9 @@
     <div>
       <h2>Privacy information</h2>
       <p>We use Matomo hosted on a little Raspberry Pi for analytics.
```

Comment:
I would change that text too, emphasizing that these are managed ourselves

Reply:
what should I put instead?

Reply:
we manage our own analytics with matoto or smth like this